### ISSUE TITLE
Abnormal Estradiol/testosterone ratio

### ISSUE THREAD

**Issue by carolina-f** [obophenotype/human-phenotype-ontology#11365]
**Preferred term label:**

**Synonyms**

**Definition (free text, please give PubMed ID)**
 PMID: 34538723  **The estradiol (pg/ml)/testosterone (ng/ml) ratio**, as a reflection of aromatization, becomes high in many cases, but not all: the ratio is >10 in 75% of AEXS cases

PMID: 22024975 In men with low fertility associated with a **decreased testosterone:estradiol ratio**, for example, in Klinefelter syndrome and obesity-associated hypogonadotropic hypogonadism, aromatase inhibitors have a positive effect.

PMID: 34538723 Blood E2 levels are elevated or within the normal range, whereas T levels are low-normal or decreased. Consequently, **E2/testosterone ratios** are invariably elevated in adult patients.
In addition, blood E1 levels were elevated in all male patients examined.

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
Aromatase excess syndrome (AEXS) ORPHA:178345

**Your nano-attribution (ORCID)**
0009-0005-6714-5727

**Comment by pnrobinson** [comment]
@carolina-f 
Adding new term: 
- ID: HP:6001443
- label: Abnormal testosterone: estradiol ratio
- parents: [Abnormal circulating sex hormone concentration (HP:0033799)](https://hpo.jax.org/app/browse/term/HP:0033799)
- pmid: PMID:39344113
- def: Any deviation from the normal ratio of testosterone and estradiol (the primary sex steroids) to each other. The testosterone:estradiol ratio is derived from total testosterone in ng/dL and total estradiol in pg/mL, and expressed as T:E. Normal ranges are affected by sex, age, and other factors.
- comment: There is a natural balance between the major sex steroids, testosterone and estradiol, controlled by gonadal secretion and peripheral conversion by aromatase. This balance is impacted by a variety of inborn and acquired conditions.
- orcid: orcid:0009-0005-6714-5727
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/11365
- synonyms:
